### ISSUE TITLE
3.2.2 framework won't build

### ISSUE THREAD

**Issue by veeamd** [Appboy/appboy-ios-sdk#131]
* [x] I have checked [Appboy's documentation](https://www.appboy.com/documentation/iOS/) and am still experiencing the issue.

# Report

## Describe your environment.

 Info                    | Value                               |
-------------------------|-------------------------------------|
 Platform Name           | ios
 Platform Version        | 11.0
 SDK Version             | 3.2.2
 Integration Method      | carthage
 Xcode Version           | Xcode 9.1
 Repro rate              | all the time (100%) 
 
## What did you do?

`carthage update --platform ios`. carthage fetches the latest 3.2.2. Build project

## What did you expect to happen?

XCode builds successfully

## What happened instead?

Xcode reports error:
```
ld: warning: ignoring file /Users/weiz/Dev/iOS/StreetEasy/Carthage/Build/iOS/SDWebImage.framework/SDWebImage, missing required architecture x86_64 in file /Users/weiz/Dev/iOS/StreetEasy/Carthage/Build/iOS/SDWebImage.framework/SDWebImage (2 slices)
ld: warning: ignoring file /Users/weiz/Dev/iOS/StreetEasy/Carthage/Build/iOS/Appboy_iOS_SDK.framework/Appboy_iOS_SDK, missing required architecture x86_64 in file /Users/weiz/Dev/iOS/StreetEasy/Carthage/Build/iOS/Appboy_iOS_SDK.framework/Appboy_iOS_SDK (2 slices)
ld: warning: Auto-Linking supplied '/Users/weiz/Dev/iOS/StreetEasy/Carthage/Build/iOS/FLAnimatedImage.framework/FLAnimatedImage', missing required architecture x86_64 in file /Users/weiz/Dev/iOS/StreetEasy/Carthage/Build/iOS/FLAnimatedImage.framework/FLAnimatedImage (2 slices)
Undefined symbols for architecture x86_64:
  "_OBJC_CLASS_$_Appboy", referenced from:
      objc-class-ref in SEAppboyTracker.o
  "_OBJC_CLASS_$_SDWebImageManager", referenced from:
      objc-class-ref in SEListing+SpotlightSearchable.o
  "_OBJC_CLASS_$_SDImageCache", referenced from:
      objc-class-ref in SEAppDelegate.o
  "_OBJC_CLASS_$_SDWebImageDownloader", referenced from:
      objc-class-ref in SEAnalyticsMediator.o
  "_OBJC_CLASS_$_SDWebImagePrefetcher", referenced from:
      objc-class-ref in SEFullScreenPhotoViewController.o
ld: symbol(s) not found for architecture x86_64
clang: error: linker command failed with exit code 1 (use -v to see invocation)
```

## Steps to reproduce

- `carthage update --platform ios`. carthage fetches the latest 3.2.2. 
- Build project

**Comment by Wenzhi** [comment]
Hey @veeamd,

I wonder if it's because that the framework isn't a fat framework and doesn't have the support for simulators. Do you have the same issue when running on a real device?

Thanks,
Wenzhi

**Comment by morgan-legal** [comment]
Hello, 

I do run into the same above issues when building on simulators.  I think there's indeed an issue with the supported architectures. Will you be able to fix this any time soon ?

Also, in the Appboy's (Braze) documentation (install via Carthage) it would be good to mention that we need to add `FLAnimatedImage.framework` & `SDWebImage.framework` for the framework to work properly. It will crash at the init otherwise.

Thanks !
Morgan

**Comment by Wenzhi** [comment]
@morgan-legal, 

Thanks for the suggestions! We will update the documentation for Carthage integration.

@veeamd and @morgan-legal, we are working on the issue and plan on releasing the fix before tomorrow morning. I'll update this issue thread as soon as the release is out.

Thanks for reporting the issue!

Best,
Wenzhi

**Comment by chshapiro** [comment]
Hey @veeamd and @morgan-legal,

We just released version 3.2.3 of the Appboy-iOS-SDK, which is a universal framework.

Please let us know if you have any further issues.

Thanks,
Chava

**Comment by morgan-legal** [comment]
It works like a charm! 
Thank you for your quick support!

Morgan
